plugins/pulp_rpm/plugins/importers/yum/pulp_solv.py

```diff
@@ -701,10 +712,12 @@ class Solver(object):
         ids.TYPE_ID_MODULEMD_DEFAULTS: module_defaults_unit_to_solvable,
     }
 
-    def __init__(self, source_repo, target_repo=None, conservative=False, ignore_missing=True):
+    def __init__(self, source_repo, target_repo=None, additional_repos=None,
+                 conservative=False, ignore_missing=True):
         super(Solver, self).__init__()
-        self.source_repo = source_repo
-        self.target_repo = target_repo
+        self.primary_source_repo = source_repo
+        self.primary_target_repo = target_repo
+        self._additional_repo_mapping = {}
```

Comment:
Does it make sense to set it to the value of `additional_repos` parameter and maybe make a default value of `additional_repos` an empty dict?

Reply:
...yeah, whoops.